airflow/contrib/operators/gcp_tasks_operator.py

```diff
@@ -0,0 +1,939 @@
+            retry=self.retry,
+            timeout=self.timeout,
+            metadata=self.metadata,
+        )
+
+
+class CloudTasksUpdateQueueOperator(BaseOperator):
+    """
+    Updates a queue in Cloud Tasks.
+
+    :param task_queue: The task queue to update.
+        This method creates the queue if it does not exist and updates the queue if
+        it does exist. The queue's name must be specified.
+    :type task_queue: dict or class google.cloud.tasks_v2.types.Queue
+    :param project_id: (Optional) The ID of the  GCP project that owns the Cloud Tasks.
+        If set to None or missing, the default project_id from the GCP connection is used.
+    :type project_id: str
+    :param location: (Optional) The location name in which the queue will be updated.
+        If provided, it will be used to construct the full queue path.
+    :type location: str
+    :param queue_name: (Optional) The queue's name.
+        If provided, it will be used to construct the full queue path.
+    :type queue_name: str
+    :param update_mask: A mast used to specify which fields of the queue are being updated.
+        If empty, then all fields will be updated.
+        If a dict is provided, it must be of the same form as the protobuf message.
+    :type update_mask: dict or class google.cloud.tasks_v2.types.FieldMask
+    :param retry: (Optional) A retry object used to retry requests.
+        If None is specified, requests will not be retried.
+    :type retry: google.api_core.retry.Retry
+    :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+        to complete. Note that if retry is specified, the timeout applies to each
+        individual attempt.
+    :type timeout: float
+    :param metadata: (Optional) Additional metadata that is provided to the method.
+    :type metadata: sequence[tuple[str, str]]]
+    :param gcp_conn_id: (Optional) The connection ID used to connect to Google Cloud Platform.
+    :type gcp_conn_id: str
+    :rtype: google.cloud.tasks_v2.types.Queue
+    """
+
+    template_fields = (
+        "task_queue",
+        "project_id",
+        "location",
+        "queue_name",
+        "update_mask",
+        "gcp_conn_id",
+    )
+
+    @apply_defaults
+    def __init__(
+        self,
+        task_queue,
+        project_id=None,
+        location=None,
+        queue_name=None,
+        update_mask=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+        gcp_conn_id="google_cloud_default",
+        *args,
+        **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.task_queue = task_queue
+        self.project_id = project_id
+        self.location = location
+        self.queue_name = queue_name
+        self.update_mask = update_mask
+        self.retry = retry
+        self.timeout = timeout
+        self.metadata = metadata
+        self.gcp_conn_id = gcp_conn_id
+        self._hook = CloudTasksHook(gcp_conn_id=self.gcp_conn_id)
```

Comment:
```suggestion
        self._hook = CloudTasksHook(gcp_conn_id=self.gcp_conn_id)
```
I think, you should create hook in execute. Otherwise, templates will not work.